--- origsrc/gnome-python-extras-2.25.3/configure.ac	2009-02-15 16:20:23.000000000 -0600
+++ src/gnome-python-extras-2.25.3/configure.ac	2009-04-21 13:14:08.686875000 -0500
@@ -81,6 +81,8 @@ AM_CONDITIONAL(OS_WIN32, test "$os_win32
 AC_DISABLE_STATIC
 AC_LIBTOOL_WIN32_DLL
 AC_PROG_LIBTOOL
+dnl when using libtool 2.x create libtool early, because it's used in configure
+m4_ifdef([LT_OUTPUT], [LT_OUTPUT])
 
 dnl check for python
 AM_PATH_PYTHON(2.3)
