--- origsrc/gnome-python-extras-2.25.3/docs/gtkmozembed/Makefile.am	2008-12-11 19:06:29.000000000 -0600
+++ src/gnome-python-extras-2.25.3/docs/gtkmozembed/Makefile.am	2009-04-21 14:10:23.686875000 -0500
@@ -53,7 +53,7 @@ build_stamp: $(REFERENCE_DEPS)builddate.
 		 --path $(BUILDDIR):$(srcdir)/gtkmozembed \
                  --stringparam gtkdoc.bookname "pygtkmozembed" \
                  --stringparam gtkdoc.version ${REF_VERSION} \
-                 $(HTML_STYLE) $(srcdir)/gtkmozembed/gtkmozembed-classes.xml
+                 $(HTML_STYLE) $(srcdir)/gtkmozembed-classes.xml
 	$(PYGOBJECT_FIXXREF) -i /usr/share/gtk-doc/html/pygtk/ $(BUILDDIR)/html/
 	touch $@
 endif
--- origsrc/gnome-python-extras-2.25.3/docs/gtkspell/Makefile.am	2008-12-11 19:06:29.000000000 -0600
+++ src/gnome-python-extras-2.25.3/docs/gtkspell/Makefile.am	2009-04-21 14:08:41.093125000 -0500
@@ -53,7 +53,7 @@ build_stamp: $(REFERENCE_DEPS)builddate.
 		 --path $(BUILDDIR):$(srcdir)/gtkspell \
                  --stringparam gtkdoc.bookname "pygtkspell" \
                  --stringparam gtkdoc.version ${REF_VERSION} \
-                 $(HTML_STYLE) $(srcdir)/gtkspell/gtkspell-classes.xml
+                 $(HTML_STYLE) $(srcdir)/gtkspell-classes.xml
 	$(PYGOBJECT_FIXXREF) -i /usr/share/gtk-doc/html/pygtk/ $(BUILDDIR)/html/
 	touch $@
 endif
